sign_verify_file_ssh.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Usage message
+usage() {
+    echo "Usage: $0 sign <file> <private_key> | verify <file> <public_key> [signature_file]"
+    exit 1
+}
+
+# Ensure at least two arguments are provided
+if [ "$#" -lt 3 ]; then
+    usage
+fi
+
+MODE="$1"
+FILE_TO_SIGN="$2"
+
+# Ensure the file exists
+if [ ! -f "$FILE_TO_SIGN" ]; then
+    echo "Error: File '$FILE_TO_SIGN' not found."
+    exit 1
```

Comment:
Maybe named constants for exit values (defined/declared at the beginning) would be nice.

Reply:
I made a stab at this but they are a bit useless unless they are somehow consistent across scripts